LibSWBF2/Chunks/LVL/lght/lght.SCOP.h

```diff
@@ -51,13 +47,35 @@ struct LIBSWBF2_API SCOP_LGHT : public GenericChunk<"SCOP"_m>{
 	void WriteToStream(FileWriter& stream) override;
 	void ReadFromStream(FileReader& stream) override;
 
+	/*
+	Local light fields
+	*/
+
 	DATA_VEC4 *p_RotationChunk;
     DATA_VEC3 *p_PositionChunk;
-    DATA_LIGHTTYPE *p_TypeChunk;
+    DATA_FLOAT *p_TypeChunk;
     DATA_VEC3 *p_ColorChunk;
 
     DATA_FLOAT *p_RangeChunk;
     DATA_VEC2 *p_ConeChunk;
```

Comment:
```DATA_LIGHTTYPE``` replaced by the more general ```DATA_FLOAT```, which also covers the Range field found on Omni and Spot lights.  ```p_ConeChunk``` is for spot lights only.